src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Util/PermissionsUtils.cs

```diff
@@ -37,6 +38,7 @@ public static async Task<ReleasePermissions> GetReleasePermissions(IUserService
             CanCreateMethodologies = await userService.CheckCanCreateMethodologyForPublication(publication).IsRight(),
             CanManageExternalMethodology =
                 await userService.CheckCanManageExternalMethodologyForPublication(publication).IsRight(),
+            CanUpdateContact = await userService.CheckCanUpdatePublication(publication).IsRight(),
```

Comment:
We already check `userService.CheckCanUpdatePublication` above, so we should consolidate this into one call to avoid an additional database query.